vtex-apps/return-app
# feat: unify schemas properties and indexes

What this PR changes:
Unify schema properties and indexes so as not to have to update each client after deployment.

How to test it:
[this workspace](https://schemas--vtexspain.myvtex.com/account#/my-returns)
Create a return request and see if it appears in the list with its corresponding values.
